python-regression/tests/features/machine6/config.yml

```diff
@@ -0,0 +1,56 @@
+default_args: &args
+  ['--testnet-coordinator',
+   'EFPNKGPCBXXXLIBYFGIGYBYTFFPIOQVNNVVWTTIYZO9NFREQGVGDQQHUUQ9CLWAEMXVDFSSMOTGAHVIBH',
+   '--mwm',
+   '1',
+   '--milestone-start',
+   '0',
+   '--testnet-no-coo-validation',
+   'true',
+   '--testnet',
+   'true',
+   '--snapshot',
+   './snapshot.txt',
+   '--local-snapshots-pruning-delay',
+   '10000'
+  ]
+
+default_ixi: &ixi
+  ['https://github.com/iotaledger/ixi-regression-tests/raw/master/tar-files/LocalSnapshots.ixi.tar.gz']
```

Comment:
Change this to a location relative to `python-regression` folder. In this case you would simply change it to `IXI/LocalSnapshot.ixi`.